docs/pages/reference/cli.mdx

```diff
@@ -1973,6 +1973,40 @@ $ tctl status \
     --identity=identity.pem
 ```
 
+### tctl alerts create
+
+Creates cluster alerts
```

Comment:
```suggestion
Creates cluster alerts.
```

Reply:
I think we should add a sentence about what a cluster alert is for users who are skimming the `tctl` reference for all available commands.